share/revng/test/configuration/revng/runtime.yml

```diff
@@ -6,7 +6,7 @@ commands:
   - type: revng.qemu-run
     from:
       - type: revng-qa.compiled-with-debug-info
-        filter: for-runtime and cross-compiler
+        filter: (for-runtime or abi-describe-functions) and cross-compiler
```

Comment:
Can you describe what was the problem in the commit message?

Reply:
Of course.

Reply:
What do you think?

[Testing: fix revng.qemu-run filter](https://github.com/Cvelth/revng/commit/dd37607aab8ce554a8b304c0453b00630517d471)
```
To avoid mangling with unrelated tests, It's detrimental to tag
`abi-describe-functions` as `for-runtime`. But they still need to be
treated similarly, hence I think it's natural to explicitly allow them
to be treated by the `revng-qa.compiled-with-debug-info` in the same
way the usual `for-runtime` tests are treated.
```

Reply:
Why is it detrimental to tag them as `for-runtime`? Does it trigger other tests we're not interested in?

If so, maybe we should split `for-runtime` instead of adding an `or` here.

Reply:
> Does it trigger other tests we're not interested in?

Yep :(

> If so, maybe we should split `for-runtime` instead of adding an `or` here.

This works just as well. Any ideas about how the thing that's currently named `for-runtime` should be called instead?

Reply:
> > Does it trigger other tests we're not interested in?
> 
> Yep :(

Which ones? Comparing output of `qemu-$ARCH` and the translated version of the binary?
If so, we can stick with `for-runtime` for those that we just want to execute but we don't care about translation + output comparison and have a new tag (which implies `for-runtime`) for translating + comparison, like `for-translation-output-comparison`.

Reply:
> Which ones?

Don't remember specifically. The thought process was "oh, wait, this is not supposed to be triggered - I probably need a separate tag".

> `for-translation-output-comparison`

Works.

Reply:
Done.